### PR TITLE
Expand Simulation landing page explanations

### PR DESCRIPTION
**Pull request by Hwurzburg**

### CODE REVIEW

File: dev/source/docs/simulation-2.rst [hunk 1]
```diff
@@ -6,11 +6,26 @@ Simulation
 
 .. image:: ../images/simulation-landingpage.jpg
 
-Simulation allows for safe testing of experimental code and settings and crashing virtual vehicles is a lot cheaper than crashing real ones!
+Simulation allows for safe testing of experimental code and settings, and crashing virtual vehicles is a lot cheaper than crashing real ones!
```
Comment (peterbarker) [inline dev/source/docs/simulation-2.rst:9]:
```suggestion
Simulation allows for safe testing of experimental code and settings - crashing virtual vehicles is a lot cheaper than crashing real ones!
```

File: dev/source/docs/simulation-2.rst [hunk 2]
```diff
@@ -6,11 +6,26 @@ Simulation
...
+
+Overview
+========
+
+Simulation is implemented by using a Flight Dynamics Model (FDM) of the vehicle to simulate the physics involved with vehicle movement, which receives inputs from a SITL (Software in the Loop) program running the ArduPilot firmware (which are the firmware's servo/motor outputs) and outputs vehicle status,position, velocities,etc. that result from those inputs back to the firmware sim just like sensors would in the real world.
```
Comment (peterbarker) [inline dev/source/docs/simulation-2.rst:14]: This is a bit of a run-on sentence.  Probably pay to break it up.

File: dev/source/docs/simulation-2.rst [hunk 3]
```diff
@@ -6,11 +6,26 @@ Simulation
...
+
+Pilot control is implemented either by joystick/sim controller box or by MAVLink commands from an attached and monitoring Ground Control Station program (GCS), like MAVProxy or Mission Planner.
```
Comment (peterbarker) [inline dev/source/docs/simulation-2.rst:16]: What's a "sim controller box"?

Comment (peterbarker) [inline dev/source/docs/simulation-2.rst:16]:
```suggestion
Pilot control is implemented either by joystick/sim controller box or by MAVLink commands from a Ground Control Station program (GCS), like MAVProxy or Mission Planner.
```

File: dev/source/docs/simulation-2.rst [hunk 4]
```diff
@@ -6,11 +6,26 @@ Simulation
...
+
+This is illustrated below:
+
+.. image:: ../images/SITL.png
+
+ArduPilot provides a native firmware simulator + FDM program (sim_vehicle.py) + graphical visualizer, simply referred to as ``SITL``, which is usually used with its developer GCS, Mavproxy.
```
Comment (peterbarker) [inline dev/source/docs/simulation-2.rst:22]:
```suggestion
ArduPilot provides a native firmware simulator + FDM program (sim_vehicle.py) + graphical visualizer, often simply referred to as ``SITL``, which is usually used with its developer GCS, Mavproxy.
```

Sometimes the MissionPlanner use of SITL is just referred to as "SITL" too :+1:

File: dev/source/docs/simulation-2.rst (hunk 3 above)
Comment (Hwurzburg) [reply dev/source/docs/simulation-2.rst:16]: Like the Interlink controller

File: dev/source/docs/simulation-2.rst (hunk 4 above)
Comment (khancyr) [inline dev/source/docs/simulation-2.rst:22]: + FDM program (sim_vehicle.py) : Nope ! sim_vehicle.py is just a launcher, the FDM is SITL default backend

File: dev/source/docs/simulation-2.rst (hunk 3 above)
Comment (Hwurzburg) [reply dev/source/docs/simulation-2.rst:16]: clarified

Comment (Hwurzburg) [reply dev/source/docs/simulation-2.rst:16]: done

File: dev/source/docs/simulation-2.rst (hunk 4 above)
Comment (Hwurzburg) [reply dev/source/docs/simulation-2.rst:22]: reworded and expanded

Comment (Hwurzburg) [reply dev/source/docs/simulation-2.rst:22]: fixed

File: dev/source/docs/simulation-2.rst (hunk 2 above)
Comment (Hwurzburg) [reply dev/source/docs/simulation-2.rst:14]: broken up

File: dev/source/docs/simulation-2.rst (hunk 1 above)
Comment (Hwurzburg) [reply dev/source/docs/simulation-2.rst:9]: changed to two statemens

File: dev/source/docs/simulation-2.rst [hunk 5]
```diff
@@ -6,11 +6,26 @@ Simulation
 
 .. image:: ../images/simulation-landingpage.jpg
 
-Simulation allows for safe testing of experimental code and settings and crashing virtual vehicles is a lot cheaper than crashing real ones!
+Simulation allows for the safe testing of experimental code and settings. Crashing virtual vehicles is a lot cheaper than crashing real ones!
+
+Overview
+========
+
+Simulation is implemented by using a Flight Dynamics Model (FDM) of the vehicle to simulate the physics involved with vehicle movement. It receives inputs from a SITL (Software in the Loop) program running the ArduPilot firmware (which are the firmware's servo/motor outputs) and outputs vehicle status,position, velocities,etc. that result from those inputs back to the firmware simulation. Just as sensors would in the real world case.
```
Comment (TunaLobster) [inline dev/source/docs/simulation-2.rst:14]: Missing a space before "etc."

File: dev/source/docs/simulation-2.rst [hunk 6]
```diff
@@ -6,11 +6,26 @@ Simulation
...
+
+Pilot control is implemented either by joystick, dedicated simulation controller box(like Interlink), or by MAVLink commands from a Ground Control Station program (GCS), like MAVProxy or Mission Planner.
+
+This is illustrated below:
+
+.. image:: ../images/SITL.png
+
+ArduPilot provides a native firmware simulator + FDM program + graphical visualizer (all launched by sim_vehicle.py), often simply referred to as ``SITL``, which is usually used with its developer GCS, Mavproxy. ArduPilot's firmware simulator and FDM can also be used by Mission Planner for SITL simulations.
```
Comment (TunaLobster) [inline dev/source/docs/simulation-2.rst:22]: Capitalization on MAVProxy. I'm not sure what native "graphical visualizer" is referring to. Flight Gear? MAVProxy GUI modules?

File: dev/source/docs/simulation-2.rst (hunk 5 above)
Comment (Hwurzburg) [reply dev/source/docs/simulation-2.rst:14]: fixxed

File: dev/source/docs/simulation-2.rst (hunk 6 above)
Comment (Hwurzburg) [reply dev/source/docs/simulation-2.rst:22]: yeah, the gui is really part of MAVProxy...clarified

**Comment by Hwurzburg** [comment]
All comments addressed....merging..thanks for the reviews

**Pull request merged**